src/data/accessibility-quiz.ts

```diff
@@ -529,7 +529,37 @@ const accessibilityQuiz = [
     Explanation:
       "The four broad guidelines of web accessibility are Perceivable, Operable, Understandable, Robust. They help ensure that web content is accessible to all users, including those with disabilities",
     Link: "https://www.freecodecamp.org/news/what-is-website-accessibility-18ce00ec990f/"
-  }
+  },
+   {
+    Question: "What is web accessibility?",
+    Answer: "the practice of building websites that are usable by everyone",
+    Distractor1: "a new JavaScript framework",
+    Distractor2: "a sorting algorithm",
+    Distractor3: "a type of programming paradigm",
+    Explanation:
+      "Web accessibility means building websites that are usable by as many people as possible.",
+    Link: "https://www.freecodecamp.org/news/what-is-website-accessibility-18ce00ec990f/",
+  },
+  {
+    Question: "Which of the following is NOT the way to fix web accessibility errors?",
+    Answer: "Changing document color",
+    Distractor1: "Updating Low Contrast Text",
+    Distractor2: "Adding Missing Alternative Text for Images",
+    Distractor3: "Fix Empty Links",
+    Explanation:
+      "Fix Empty Links, Add Missing Alternative Text for Images and Update Low Contrast Text are the common ways to fix the accessiblility errorf",
```

Comment:
fixing spelling errors and grammar issues 

```suggestion
      "Fixing empty links, adding missing alt text for images and updating low contrast text are some common ways to fix the accessibility errors",
```